build.gradle

```diff
@@ -52,20 +54,82 @@ allprojects {
       target '**/*.md'
       targetExclude 'docs/release-notes.md'
       endWithNewline()
-      // Disabling Prettier since it causes TravisCI to barf
-      // prettier()
     }
   }
+
+  jacoco {
+    toolVersion = "0.8.7"
+  }
 }
 
+ext.moduleCoverageThresholds = [
+  // Default coverage applied unless overridden
+  'coral-dbt': 0.95,
+  'coral-incremental': 0.95,
+  'coral-pig': 0.85,
+  'coral-schema': 0.80,
+  'coral-service': 0.95,
+  'coral-spark': 0.90,
+  'coral-spark-plan': 0.74,
+  'coral-trino': 0.80,
+  'coral-visualization': 0.75,
```

Comment:
I'm assuming these thresholds are more or less just arbitrary?

Reply:
I came down to these values based on their existing test coverage in the codebase, and set the threshold a bit below the current coverage to make sure it won't cause flaky immediate build failures (this coverage not passing does fail build now, from past experience the coverage might have very slight difference depending on different execution environment)